roles/confluent.variables/vars/main.yml

```diff
@@ -1115,3 +1115,248 @@ control_center_final_properties: "{{ control_center_combined_properties | combin
 
 ### The base path for the binary files. When in Archive File deployment mode this results in binary files being based in, for example `/opt/confluent/confluent-5.5.1/bin`, otherwise they are based in `/usr/bin`.
 binary_base_path: "{{ archive_config_base_path+'/confluent-'+confluent_package_version if installation_method == 'archive' else '/usr' }}"
+
+#### Kafka Connect Replicator Variables ####
+kafka_connect_replicator_service_name: kafka-connect-replicator
+kafka_connect_replicator_packages:
+  - confluent-common
+  - confluent-rest-utils
+  - confluent-metadata-service
+  - confluent-ce-kafka-http-server
+  - confluent-kafka-rest
+  - confluent-server-rest
+  - confluent-telemetry
+  - "{{ kafka_broker_main_package }}"
+  - confluent-hub-client
+  - confluent-kafka-connect-replicator
+  - confluent-security
+  - confluent-rebalancer
+  - confluent-control-center-fe
+  - confluent-control-center
+  - confluent-schema-registry
+
+kafka_connect_replicator_default_user: cp-kafka-connect-replicator
+kafka_connect_replicator_default_group: confluent
+kafka_connect_replicator_default_log_dir: /var/log/confluent/
+kafka_connect_replicator_default_log_name: kafka-connect-replicator.log
+kafka_connect_replicator_max_log_files: 10
+kafka_connect_replicator_log_file_size: 100mb
+
+kafka_connect_replicator:
+  server_start_file: "{{ binary_base_path }}/bin/kafka-connect-replicator-start"
+  replication_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator.properties"
+  consumer_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator-consumer.properties"
+  producer_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator-producer.properties"
+  interceptors_config_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka-connect-replicator/kafka-connect-replicator-interceptors.properties"
+  systemd_file: /lib/systemd/system/{{kafka_connect_replicator_service_name}}.service
+  systemd_override: /etc/systemd/system/{{kafka_connect_replicator_service_name}}.service.d/override.conf
+  log4j_file: "{% if installation_method == 'archive' %}{{archive_destination_path}}/confluent-{{confluent_package_version}}{% endif %}/etc/kafka-connect-replicator/replicator-log4j.properties"
+  cluster_id: replicator
```

Comment:
if the cluster.id is meant to be configurable it should live in defaults not vars and be documented